livingatlas/pipelines/src/test/java/au/org/ala/parser/DistanceRangeParserTest.java

```diff
@@ -30,21 +32,19 @@ public void parseRange() {
     value = DistanceRangeParser.parse("100km,");
     assertEquals(100000f, value, 0);
 
-    value = DistanceRangeParser.parse(" 1km-20km,");
+    value = DistanceRangeParser.parse(" 1kilometers-20kilometres");
```

Comment:
why has the original test been replaced ? shouldnt we just have both tests ?
Similarly for the other examples....

Reply:
Since there is a test for km, but no harm to add it back